diff --git a/PKGBUILD b/PKGBUILD
index 59a1010..9a6fd15 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -2,7 +2,7 @@
 # Contributor: ianux <ianux@free.fr>

 pkgname=weboob-git
-pkgver=1.0-819-g925beee
+pkgver=55fa318c
 pkgrel=1
 pkgdesc="Weboob (Web Out Of Browsers) provides several applications to interact with a lot of websites."
 url="http://weboob.org"
@@ -38,15 +38,9 @@ optdepends=('gnupg: check for repository authenticity'
             'python2-nose: test suite')
 conflicts=('weboob')
 provides=('weboob')
-source=("$pkgname"::'git://git.symlink.me/pub/weboob/devel.git')
+source=("$pkgname"::"git://git.symlink.me/pub/weboob/devel.git#commit=$pkgver")
 sha256sums=('SKIP')

-pkgver() {
-    cd ${srcdir}/${pkgname}
-    local ver="$(git describe --long)"
-    printf "%s" "${ver//-/.}"
-}
-
 package() {
    cd ${srcdir}/${pkgname}
    python2 setup.py install --qt --xdg --prefix=/usr --root="$pkgdir"
